book/src/ui/interacting.md

```diff
@@ -0,0 +1,159 @@
+# Interaction!
+
+Now that we have a button we can interact with, let's see how we actually do this.
+We will show you how to do this in two ways. One way will be interaction through a system,
+and the latter interaction through `handle_event` method of your active state.
+
+## Creating the system
+
+Let's start of with some boilerplate code: 
+
+```rust 
+pub struct SimpleButtonSystem;
+
+impl System for SimpleButtonSystem {
+	type SystemData = ();
+
+	fn run(&mut self, data: Self::SystemData) {
+	
+	}
+}
+```
+
+This was shown in previous chapters. The way you will be able to read the generated 
```

Comment:
Best link to `../concepts/system/system_initialization.html`. It makes it easier for a reader to know where to look.